### DISCUSSION TITLE
Docker volume patch should be in the stack directory

### DISCUSSION THREAD

**Discussion by flotpg** [louislam/dockge#91] [Ask for Help] [answered]
Hi,
thanks a lot for this awesome app!

I expected the volumes created inside the stack directory instead of /var/lib/docker/volumes

Best regards, Flo.

**Comment by TBT-TBT** [comment, marked as answer]
Then you have to stop Docker itself, change the Docker volume location in deamon.json , move all volumes to the new location and start Docker again.

That is nothing Dockge has to touch, as it is a basic Docker setting.
